idea/idea-core/src/org/jetbrains/kotlin/idea/core/psiModificationUtils.kt

```diff
@@ -92,6 +95,14 @@ fun KtLambdaArgument.moveInsideParenthesesAndReplaceWith(
     return oldCallExpression.replace(newCallExpression) as KtCallExpression
 }
 
+private fun shouldLambdaParameterNamed(args: List<ValueArgument>, callExpr: KtCallExpression): Boolean {
+    return args.any { it.isNamed() } ||
```

Comment:
Now that it's a separate function, there's no need to cramp it all into a single expression.

```
if (args.any { it.isNamed () }) return true
val callee = ...resolve as? KtFunction ?: return true
return callee.valueParameters.any { ... }
```

Also the grammatically correct name would be `shouldLambdaParameterBeNamed`.